CHANGELOG.md

```diff
@@ -35,31 +34,40 @@ For example:
 @import "node_modules/govuk-frontend/govuk/components/header";
 ```
 
+Find out more about [importing CSS](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#css).
+
 [Pull request #1804: Allow components to be imported without dependencies](https://github.com/alphagov/govuk-frontend/pull/1804).
 
+#### Back links and breadcrumbs are easier to select
+
+The [back link](https://design-system.service.gov.uk/components/back-link/) and [breadcrumbs](https://design-system.service.gov.uk/components/breadcrumbs/) components are now bigger, so they're easier to select if you're using a touch screen.
+
+The design of the back link component is now more consistent with the breadcrumb component. 
 
-### Deprecations
+[Pull request #1753: Make back link arrow consistent with breadcrumb component](https://github.com/alphagov/govuk-frontend/pull/1753).
 
-#### Importing files from the core or overrides layers
+### Deprecated features
 
-If you import specific files from the core or overrides layers, you should import the `base` file first:
+#### Importing from the `core` and `overrides` layers
+
+If you import specific files from the `core` or `overrides` layers, you now need to import `node_modules/govuk-frontend/govuk/base` first.
+
+For example:
 
 ```scss
 @import "node_modules/govuk-frontend/govuk/base";
-
 @import "node_modules/govuk-frontend/core/typography";
 ```
 
-Importing files from the core or overrides layers without importing `base` will [no longer work as of GOV.UK Frontend v4.0](https://github.com/alphagov/govuk-frontend/issues/1800).
+If you do not import `node_modules/govuk-frontend/govuk/base` first, your service will no longer work from GOV.UK Frontend v4.0.0.
 
 [Pull request #1807: Warn if importing core, overrides without dependencies](https://github.com/alphagov/govuk-frontend/pull/1807).
 
-
```

Comment:
I find it useful to have 2 blank lines at the end of a 'section' (basically when we go 'up' a heading level, as I find it easier to spot where the section ends and another starts, but appreciate we're not doing that consistently.

```
## 3.7.0

### New features

#### Better stuff

It's all better now!

#### Faster things

It goes faster


### Fixes

- More cake!
- Less salad


## 3.6.0

### New features

...
```

Reply:
Ok - added here, but worth knowing that this does get highlighted as a markdown validation error in my editor.

Reply:
Ah, interesting. I'm not wedded to it – just personally find it useful and wanted to explain the thinking :)